### PR TITLE
PhantomJS is abandoned, replace it with Selenium/Chrome headless

### PR DESCRIPTION
**Pull request by guilleiguaran**
PhantomJS was abandoned and Chrome headless is seeing a major adoption: 

https://github.com/ariya/phantomjs/issues/15105

### CODE REVIEW

**Comment by jasonfb** [comment]
**PHANTOM JS IS DEAD. LONG LIVE PHANTOM JS**

File: ci/qunit-selenium-runner.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'qunit/selenium/test_runner'
+require 'chromedriver/helper'
+
+driver_options = Selenium::WebDriver::Chrome::Options.new
+driver_options.add_argument('--headless')
+driver_options.add_argument('--disable-gpu')
```
Comment (vemv) [inline ci/qunit-selenium-runner.rb]: worth explaining why gpu is disabled

Comment (tinogomes) [reply ci/qunit-selenium-runner.rb]: https://developers.google.com/web/updates/2017/04/headless-chrome#cli

Comment (vemv) [reply ci/qunit-selenium-runner.rb]: Thanks!

But I meant, I would have added a comment like

`# https://developers.google.com/web/updates/2017/04/headless-chrome#cli indicates this is flag needed for now`

**Pull request merged**